python/BioSimSpace/Sandpit/Exscientia/Process/_amber.py

```diff
@@ -186,7 +186,7 @@ def __init__(
         self._is_header = False
 
         # The names of the input files.
-        self._rst_file = "%s/%s.rst7" % (self._work_dir, name)
+        self._rst_file = "%s/%s.rst" % (self._work_dir, name)
```

Comment:
This will be painful to visualise in PyMOL now...

Reply:
Is it worth still keeping the extension to `rst7`? As far as I know, `rst` and `rst7` are the same format, it's just that it can be either binary ot ASCII.

Reply:
Yes. But I guess there is no other easy way to fix it.